documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -501,27 +501,12 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Examples**:
 ```json
-"parameters": {
-    "ApiVersionParameter": {
-      "name": "x-ms-version",
-      "x-ms-client-name": "version",
-      "in": "header",
-      "required": false,
+  "headers": {
+    "x-ms-request-id": {
+      "x-ms-client-name": "requestId",
```

Comment:
I think you can still use a parameter definition,  actually your example is a response header schema , it's not a parameter which does not meet the linting rule.

Reply:
Agreed that this should be a parameter.  I've pushed a new commit with a better example.